index.html

```diff
@@ -289,17 +289,14 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
     <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
 							<img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
-		                    <img class="col-sm-2" alt="Chahat Bansal" src="img/chahat.png">
+							<img class="col-sm-2" alt="Chahat Bansal" src="img/chahat.png">
```

Comment:
Properly intend these lines.

Reply:
I haven't indented this. This is from very earlier.

Reply:
Please see the amount of additional spaces added in the giff. If you have done it, then it must be your editor, pls avoid ide's